specs.md

```diff
@@ -169,17 +171,31 @@ BMS-9 | Charger detection and logging | - The BMS must check the charger connect
 BMS-10 | Charger enable/disable | The BMS must enable the charger by setting the BMS PON pin high and disable the charger by setting the BMS PON pin low.
 BMS-11 | AIRs weld/stuck open detection | The BMS must check that the AIR+ and AIR- are in the desired open or closed state at 100Hz, and if not the BMS must throw an AIR shutdown fault and enter the fault state.
 BMS-36 | IMD data transmission | The BMS must transmit the high/low status of the IMD's OK_HS output, the information encoded in the IMD's PWM output, and the seconds elapsed since the IMD was powered on.
-BMS-37 | OK status transmission | The BMS must transmit the on/off status of BMS_OK, IMD_OK and BSPD_OK at 100Hz or faster.
+BMS-37 | OK status transmission | The BMS must transmit the on/off status of BMS_OK, IMD_OK and BSPD_OK at 100Hz or faster, and on transition in an interrupt.
```

Comment:
After discussing with @kirons96 this makes sure we do not drop OK_Statuses when we are logging CAN signals

Reply:
This should be considered though

![image](https://user-images.githubusercontent.com/32230980/102323533-742f3700-3f35-11eb-8a7f-8012df2c1b84.png)

Reply:
isn't adding CAN messages to the CAN queue non-blocking and fine from an ISR context? i don't think we need task notifs for this, but maybe @thekenu can help

Reply:
@kirons96 Not sure either I will ask Ken. I remember seeing this from a while ago

Reply:
@thekenu B